tests/integ/scala/test_file_operation_suite.py

```diff
@@ -219,6 +222,55 @@ def test_put_negative(session, temp_stage, temp_source_directory, path1):
     assert "does not exist or not authorized." in str(stage_not_exist_info)
 
 
+def test_put_stream_with_one_file(session, temp_stage, path1, path2, path3):
+    stage_prefix = f"prefix_{random_alphanumeric_name()}"
+    stage_with_prefix = f"@{temp_stage}/{stage_prefix}"
+    file_name = os.path.basename(path1)
+    with open(path1, "rb") as fd:
+        first_result = session.file.put_stream(fd, stage_with_prefix, file_name)
+    assert first_result.source == os.path.basename(path1)
+    assert first_result.target == os.path.basename(path1) + ".gz"
+    assert first_result.source_size in (10, 11)
+    assert first_result.target_size in (32, 64)
```

Comment:
I'm unsure of how to set correct values for asserting source and target size. They passes on my mac but could change later

Reply:
hmm it would be flaky. Probably just check it's not None.

Reply:
Yeah different platforms may have different size. It almost killed me.